scripts/reportportal/rp_utils/rp_client.py

```
@@ -0,0 +1,273 @@
+
+
+class RPClient:
+    """ReportPortal REST API client with pagination support.
+
+    Args:
+        base_url: ReportPortal instance URL (e.g., "https://reportportal.example.com").
+        project: RP project name (e.g., "cnv")
+        token: Bearer token for authentication
+    """
+
+    def __init__(self, base_url: str, project: str, token: str, verify_ssl: bool = False) -> None:
```

Ruff (0.15.18)

[warning] Boolean-typed positional argument in function definition

(FBT001)